ccd-definition/CaseField.json

```diff
@@ -181,6 +181,16 @@
     "_Definition":  "The delivery method type of service from the applicant solicitor to the first respondent solicitor",
     "_Category":  "CIVIL"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "serviceNamedPersonToRespondentSolicitor1",
+    "Label": "On whom did you serve?",
```

Comment:
is it On or To?

Reply:
On apparently! Seems odd to me too 🤷 

"Who did you serve to?" sounds more correct imo

Reply:
maybe even name of person served to

Reply:
I though it was only me because not native english speaker...